common-arcana.lic

```diff
@@ -133,6 +142,28 @@ module DRCA
     DRC.retreat(settings.ignored_npcs) unless spell['skip_retreat']
   end
 
+  def prepare_to_cast_runestone(spell, settings)
+    if DRCI.inside?("#{spell['runestone_name']}", settings.runestone_storage)
+      return false if !get_runestone(spell['runestone_name'], settings)
+    else
+      DRC.message("*** Out of #{spell['runestone_name']}! ***")
+      return false
+    end
+    DRC.bput("harness #{settings.runestone_harness}", "You tap into the mana") if settings.runestone_harness
+    waitcastrt?
+    return true
+  end
+
+  def get_runestone(runestone, settings)
+    return if DRCI.in_hands?(runestone)
```

Comment:
Could you explicitly return `true` or `false` here?

Reply:
Yolu mean `return true if DRCI.in_hands?(runestone)`?

Reply:
Yes

Reply:
Added true. Testing.